packages/gateway-conformance/src/fixtures/ipns-record-datastore.ts

```diff
@@ -0,0 +1,11 @@
+import { MemoryDatastore } from 'datastore-core'
+import type { Datastore } from 'interface-datastore'
+
+const datastore = new MemoryDatastore()
+/**
+ * We need a normalized datastore so we can set custom records
+ * from the IPFS_NS_MAP like kubo does.
+ */
+export function getIpnsRecordDatastore (): Datastore {
+  return datastore
+}
```

Comment:
the idea for this was to keep ipns-record fixture loading separate from the datastore used by verified-fetch, but now it's just a singleton so that kubo-mgmt and conformance.spec.ts use the same datastore.